cmd/hypper/testdata/output/install-dry-run-with-all-optional-deps.txt

```diff
@@ -0,0 +1,5 @@
+🛳  Installing shared dependencies for chart "empty":
```

Comment:
There should be some output stating that this is a dry run IMHO.

Reply:
There is none in Helm, so I went with that.

Reply:
I do not like that there either :)

Reply:
:thinking:. To be honest I prefer my commands to be as less verbose as possible: on success, don't print anything, unless is information extracted from running the operation and not known beforehand.
In this case, the user has already passed `--dry-run`, so they know that they are doing a dry run, no need to remind them (there's no way to be safe if users don't even remember what flags and options are they passing).
Also, people use the `--dry-run`, expecting the same output as normal run, for CI and tests.

Reply:
On Helm, doing `--dry-run --debug` will print debug info informing on it. We get that for free, as we ultimately call Helm's action.Install.
It's the same behaviour from Golang utilities: `go vet`, `go build`, etc don't print any known info from passing flags, unless you make it verbose.

Reply:
:+1: